### DISCUSSION TITLE
Complex numbers?

### DISCUSSION THREAD

**Discussion by chillenb** [OpenChemistry/chemicaljson#25] [Ideas]
Sometimes orbital coefficients are complex, but JSON doesn't understand complex numbers. What do you think is a good way to deal with this?

**Comment by ghutchis** [comment]
At the moment, we've been optimizing for *most* use-cases in chemistry. Most quantum chemistry programs output real-valued orbital coefficients.

My suggestion would be to add a separate key for imaginary components, if they exist, e.g. 
      root["orbitals"]["moCoefficientsImaginary"]
 # or root["orbitals"]["alphaCoefficientsImaginary"]

I don't think QCSchema handles this case either.

**Reply by chillenb** [reply]
OK! Thanks for your suggestion; it's pretty sensible. I just wish JSON supported more numeric types.
